$OpenBSD: patch-taglib_mp4_mp4coverart_h,v 1.1 2012/11/12 11:12:03 dcoppa Exp $

Correctly parse covr atoms with type 0
(upstream git commit 3a5aeb457324f9dcac2c0051e94b0d748af63f65)

--- taglib/mp4/mp4coverart.h.orig	Mon Nov 12 10:49:36 2012
+++ taglib/mp4/mp4coverart.h	Mon Nov 12 10:50:42 2012
@@ -45,7 +45,8 @@ namespace TagLib {
         JPEG = TypeJPEG,
         PNG  = TypePNG,
         BMP  = TypeBMP,
-        GIF  = TypeGIF
+        GIF  = TypeGIF,
+        Unknown = TypeImplicit,
       };
 
       CoverArt(Format format, const ByteVector &data);
